### ISSUE TITLE
Fix duplication of variant Creutzfeld Jacob disease

### ISSUE THREAD

**Issue by paolaroncaglia** [EBISPOT/efo#580]
These are duplicates:
EFO:0004597 variant Creutzfeld Jacob disease
EFO:1001233 variant Creutzfeldt-Jakob disease
see https://github.com/monarch-initiative/mondo/issues/847

**Comment by d0choa** [comment]
Denise in our team spotted this one as well. The `EFO:1001233` version has also problematic synonyms such as `Encephalopathy, Bovine Spongiform` that look incorrect and also clashes with actual terms (MONDO:0025149).

She also suggested that the related `scrapie - EFO:1001168` should be moved to `animal disease`

**Comment by paolaroncaglia** [comment]
Pinning this issue as high priority bcs it's causing an error during 'make all', presumably due to the Mondo edit now being released:
2019-10-28 13:05:20,739 ERROR org.obolibrary.robot.ReasonOperation - Reference violation: InvalidReferenceViolation [axiom=SubClassOf(<http://www.ebi.ac.uk/efo/EFO_0004597> <http://www.ebi.ac.uk/efo/EFO_0004226>), referencedObject=<http://www.ebi.ac.uk/efo/EFO_0004597>, category=DEPRECATED]
where
EFO_0004597 = variant Creutzfeld Jacob disease
EFO_0004226 = Creutzfeldt Jacob Disease

**Comment by paolaroncaglia** [comment]
@twhetzel 
Just letting you know that an EFO term that had gwas trait = true has been obsoleted because it was a duplicate, and replaced by the other equivalent term. I merged the two and made sure that the replacement has the gwas trait = true. The two terms are
EFO:0004597 variant Creutzfeld Jacob disease http://www.ebi.ac.uk/efo/EFO_0004597 (now obsoleted)
EFO:1001233 variant Creutzfeldt-Jakob disease http://www.ebi.ac.uk/efo/EFO_1001233 (current replacement).
Any issue please let me know, thanks.

**Comment by paolaroncaglia** [comment]
Leaving this ticket open as a reminder to address the following:
- [ ] "The EFO:1001233 version has also problematic synonyms such as Encephalopathy, Bovine Spongiform that look incorrect and also clashes with actual terms (MONDO:0025149)." 
- [ ] "the related scrapie - EFO:1001168 should be moved to animal disease".

**Comment by paolaroncaglia** [comment]
@d0choa 
FYI, for EFO:1001233 'variant Creutzfeldt-Jakob disease' I fixed the incorrect synonyms (and definition) that were pointing to the bovine disease. However, one of those synonyms comes from Mondo and needs to be removed at the source. I opened a ticket with them, for that issue and for a related one (https://github.com/monarch-initiative/mondo/issues/908). 
As for your (and Denise's) comment that "the related scrapie - EFO:1001168 should be moved to animal disease", 'scrapie' is under 'prion disease', which is correct, and prion disease is a broad term that encompasses the bovine as well as the human host. I added a note in https://github.com/EBISPOT/efo/issues/584, so we may refer to that ticket for discussion.
